astropy/constants/tests/test_pickle.py

```diff
@@ -1,7 +1,10 @@
 # Licensed under a 3-clause BSD style license - see LICENSE.rst
 from __future__ import absolute_import, division, print_function, unicode_literals
+
+import pytest
+
 from ... import constants as const
-from ...tests.helper import pytest, pickle_protocol, check_pickling_recovery
+from ...tests.helper import pickle_protocol, check_pickling_recovery  # noqa
```

Comment:
The `noqa` baffled me, until I noticed the tests actually never get run. Which is strange, but outside of scope here -- see #5719